package.json

```diff
@@ -31,57 +38,45 @@
     "cdk/*.ts"
   ],
   "dependencies": {
-    "@aws-cdk/aws-cloudwatch": "~1.148.0",
-    "@aws-cdk/aws-cloudwatch-actions": "~1.148.0",
-    "@aws-cdk/aws-docdb": "~1.148.0",
-    "@aws-cdk/aws-ec2": "~1.148.0",
-    "@aws-cdk/aws-ecr": "~1.148.0",
-    "@aws-cdk/aws-ecr-assets": "~1.148.0",
-    "@aws-cdk/aws-ecs": "~1.148.0",
-    "@aws-cdk/aws-ecs-patterns": "~1.148.0",
-    "@aws-cdk/aws-elasticache": "~1.148.0",
-    "@aws-cdk/aws-elasticloadbalancingv2": "~1.148.0",
-    "@aws-cdk/aws-events": "~1.148.0",
-    "@aws-cdk/aws-events-targets": "~1.148.0",
-    "@aws-cdk/aws-iam": "~1.148.0",
-    "@aws-cdk/aws-lambda": "~1.148.0",
-    "@aws-cdk/aws-logs": "~1.148.0",
-    "@aws-cdk/aws-rds": "~1.148.0",
-    "@aws-cdk/aws-s3": "~1.148.0",
-    "@aws-cdk/aws-secretsmanager": "~1.148.0",
-    "@aws-cdk/aws-sns": "~1.148.0",
-    "@aws-cdk/aws-sns-subscriptions": "~1.148.0",
-    "@aws-cdk/aws-ssm": "~1.148.0",
-    "@aws-cdk/core": "~1.148.0",
-    "@types/node": "^15.3.0",
-    "@types/object-hash": "^2.1.0",
-    "aws-cdk": "~1.148.0",
-    "aws-sdk": "^2.983.0",
+    "@types/node": "^17.0.39",
+    "@types/object-hash": "^2.2.1",
+    "aws-cdk": "^2.28.1",
+    "aws-cdk-lib": "^2.28.1",
+    "aws-sdk": "^2.1156.0",
     "camel-case": "^4.1.2",
     "chalk": "^4.1.1",
     "commander": "^6.2.0",
-    "conf": "^10.0.1",
-    "figlet": "^1.5.0",
+    "conf": "^10.1.2",
+    "constructs": "^10.1.42",
+    "figlet": "^1.5.2",
     "flat": "^5.0.2",
-    "moment": "^2.29.1",
+    "moment": "^2.29.3",
     "node-fetch": "^2.6.1",
     "object-hash": "^3.0.0",
-    "prompts": "^2.4.1",
-    "semver": "^7.3.5",
-    "table": "^6.7.1",
+    "prompts": "^2.4.2",
+    "semver": "^7.3.7",
+    "source-map-support": "^0.5.21",
+    "table": "^6.8.0",
     "tempy": "^1.0.1",
-    "ts-node": "^9.1.1",
-    "typescript": "^4.2.4",
+    "ts-node": "^10.8.1",
+    "typescript": "^4.7.3",
```

Comment:
Is typescript really required in dependencies? I'm assuming this is built and probably typescript is a `devDependency`

Reply:
it's not. i'm not sure how those migrated out of the dev dependencies. i've moved them back.